sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/DependencyResolutionOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.iot.modelsrepository;
+
+/*
```

Comment:
Need this line to be "/**" to make the comment public to users of our library. Without it, this comment behaves the same way as a line "// some comment"